back/prisma/migrations/20231002225642_add_trusted_device_table/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- CreateTable
+CREATE TABLE "TrustedDevice" (
+    "id" SERIAL NOT NULL,
+    "ip" TEXT NOT NULL,
+    "userId" INTEGER NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "TrustedDevice_pkey" PRIMARY KEY ("ip")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "TrustedDevice_id_key" ON "TrustedDevice"("id");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "TrustedDevice_ip_key" ON "TrustedDevice"("ip");
+
+-- AddForeignKey
+ALTER TABLE "TrustedDevice" ADD CONSTRAINT "TrustedDevice_userId_fkey" FOREIGN KEY ("userId") REFERENCES "User"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
We said that when needing to add a migration we would erase the previous one in order to not replicate errors with prisma and git :(
Do you think you could do that?
Or did you find the reason why the migration failed in the first place with our workflow?

Reply:
Done!